src/components/@molecules/NFTTemplate/NFTTemplate.tsx

```diff
@@ -54,11 +54,12 @@ const Text = styled.text(
 
 const NFTTemplate = ({ name, backgroundImage, isNormalised }: Props) => {
   const Satoshi = new FontFace('Satoshi', 'url(/fonts/sans-serif/Satoshi-Bold.otf)')
-  const DejaVuSans = new FontFace('DejaVuSans', 'url(/fonts/sans-serif/DejaVuSans-Bold.ttf)')
```

Comment:
Is this here to make sure that the font face is loaded before making "elementData" calculations? If so, then they should probable be wrapped in a useEffect similar to loading the polyfill

Reply:
I think we can just move font load outside of the the component, since it has no dependency or bound with react, will be a constant.

Reply:
Based on chat discussion, we do not need to re-load the Satoshi font, since it is preloaded in the main style of the app.